minecraft/curse.gradle

```diff
@@ -30,7 +30,7 @@ curseforge {
             addGameVersion "${it}"
         }
 
-        if (System.getEnv('curseRequrements')) {
+        if (System.getEnv('curseRequirements')) {
```

Comment:
People import these scripts as pseudo plugins so this is a breaking change. Would need to add backwards compatibility.

Reply:
Alright